### PR TITLE
Update to Golang 1.13

### PR DESCRIPTION
**Pull request by boekkooi-fresh**
Update to using goland 1.13 with go modules.
This also updates the deps to there latest version which should fix some issue we have with the latest MySQL version.

### CODE REVIEW

**Comment by aleksandrzhiliaev** [comment]
+1

**Comment by Bahjat** [comment]
@boekkooi-fresh I tested with the following `docker-compose.yml`
```yml
version: "3.1"

services:
  db:
    image: mysql
    command: --default-authentication-plugin=mysql_native_password
    restart: always
    ports:
      - 3306:3306
    environment:
      MYSQL_ROOT_PASSWORD: example
```
- create empty db `bob_local_us`
- build the pr changes
```sh
$ /Users/bahjat/go/src/github.com/hellofresh/klepto/dist/klepto_darwin_amd64/klepto steal --from='api-v2-us:XXXXX@tcp(intfood-db-us000.staging.hellofresh.io:3306)/bob_staging_us' --to='root:example@tcp(localhost:3306)/bob_local_us'
```
```
• MYSQL dumper forcing multistatements! help=https://github.com/go-sql-driver/mysql#multistatements
• Stealing...              
⨯ Error while dumping       error=failed to dump structure: Error 1050: Table 'abilita_api_logs' already exists
```

**Comment by boekkooi-fresh** [comment]
@Bahjat I think you have an issue with the db you created can you try using the following docker-compose.yml and not create a db your self. Another option is that you ran the command twice without removing the already added table structures.
```yaml
version: "3.1"

services:
  db:
    image: mysql:8.0.16
    command: --default-authentication-plugin=mysql_native_password
    ports:
      - 3306:3306
    environment:
      MYSQL_ROOT_PASSWORD: example
      MYSQL_DATABASE: bob_local_us
```

I did find another issue with mismatching SQL_MODE's which I fixed in the last commit.

**Comment by Bahjat** [comment]
started to do what is expected, great @boekkooi-fresh 👍

**Comment by craig-foster** [comment]
+1

File: .travis.yml [hunk 1]
```diff
@@ -22,14 +18,16 @@ env:
 install:
   - mkdir -p $GOPATH/bin
   - go get github.com/go-playground/overalls
-  - make deps
+  - go get github.com/goreleaser/goreleaser
```
Comment (aleksandrzhiliaev) [inline .travis.yml]: why not use `make deps` here?

Comment (boekkooi-fresh) [reply .travis.yml]: Because it caused `go mod vendor` to be called as part of deps and we don't use vendored mode in travis.

**Comment by aleksandrzhiliaev** [comment]
+1

**Pull request merged**